composer.json

```
@@ -0,0 +1,50 @@
+{
+    "name": "letsgoi/laravel-settings",
+    "description": "Laravel settings package",
+    "keywords": [
+        "goi",
+        "letsgoi",
+        "laravel",
+        "settings"
+    ],
+    "homepage": "https://github.com/letsgoi/laravel-settings",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "GOI",
+            "email": "tech@letsgoi.com"
+        }
+    ],
+    "require": {
+        "php": "^8.0",
+        "illuminate/database": "^6.0|^7.0|^8.0|^9.0",
+        "illuminate/http": "^6.0|^7.0|^8.0|^9.0",
+        "illuminate/support": "^6.0|^7.0|^8.0|^9.0"
```

Actually we only need to support Laravel 8 and 9.
```suggestion
        "illuminate/database": "^8.0|^9.0",
        "illuminate/http": "^8.0|^9.0",
        "illuminate/support": "^8.0|^9.0"
```

---

Done, changed